src/sidebar/components/Excerpt.tsx

```diff
@@ -55,36 +57,52 @@ function InlineControls({ isCollapsed, setCollapsed, linkStyle = {} }) {
 
 const noop = () => {};
 
-/**
- * @typedef ExcerptProps
- * @prop {object} [children]
- * @prop {boolean} [inlineControls] - If `true`, the excerpt provides internal
- *   controls to expand and collapse the content. If `false`, the caller sets
- *   the collapsed state via the `collapse` prop.  When using inline controls,
- *   the excerpt is initially collapsed.
- * @prop {boolean} [collapse] - If the content should be truncated if its height
- *   exceeds `collapsedHeight + overflowThreshold`.  This prop is only used if
- *   `inlineControls` is false.
- * @prop {number} collapsedHeight - Maximum height of the container, in pixels,
- *   when it is collapsed.
- * @prop {number} [overflowThreshold] - An additional margin of pixels by which
- *   the content height can exceed `collapsedHeight` before it becomes collapsible.
- * @prop {(isCollapsible: boolean) => void} [onCollapsibleChanged] - Called when the content height
- *   exceeds or falls below `collapsedHeight + overflowThreshold`.
- * @prop {(collapsed: boolean) => void} [onToggleCollapsed] - When `inlineControls` is `false`, this
- *   function is called when the user requests to expand the content by clicking a
- *   zone at the bottom of the container.
- * @prop {object} [settings] - Used for theming.
- */
+type ExcerptProps = {
+  children?: ComponentChildren;
+  /**
+   * If `true`, the excerpt provides internal
```

Comment:
The line length/wrapping for these docstring comments got ragged in conversion here. Can you re-adjust?